contrib/exporters/secadvisor/mod/acc_rej_action.go

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"time"
+
+	"github.com/skydive-project/skydive/common"
+	"github.com/skydive-project/skydive/flow"
+	"github.com/spf13/viper"
+)
+
+type FlowInfoStatus int32
+
+const (
+	FlowInfoOneEntry   FlowInfoStatus = 1
+	FlowInfoTwoEntries FlowInfoStatus = 2
+	FlowInfoOldFlow    FlowInfoStatus = 3
+)
+
+// time for cleanup of old connections
+const timeoutPeriod = 20
+
+type accRejFlowInfo struct {
+	Status                   FlowInfoStatus
```

Comment:
redundant: you can remove Status field and use `Flow{1,2}` == or ~= `nil`

Reply:
Flow2 will not be nil because it will be set from the previous iteration. This information will be needed to produce the accurate accounting when a stream is blocked after it was previously accepted.